helm-service/controller/onboarder.go

```diff
@@ -124,18 +124,18 @@ func (o *Onboarder) DoOnboard(ce cloudevents.Event, loggingDone chan bool) error
 
 func (o *Onboarder) checkAndSetServiceName(event *keptnevents.ServiceCreateEventData) error {
 
-	errorMsg := "Service name contains upper case letter(s) or special character(s).\n " +
-		"Keptn relies on the following conventions: " +
-		"start with a lower case letter, then lower case letters, numbers, and hyphens are allowed."
-
 	if event.HelmChart == "" {
 		// Case when only a service is created but not onboarded (i.e. no Helm chart is available)
-		if !keptnutils.ValidateKeptnEntityName(event.Service) {
-			return errors.New(errorMsg)
+		if !keptnutils.ValididateUnixDirectoryName(event.Service) {
+			return errors.New("Service name contains special character(s)")
```

Comment:
Same as with the CLI, please add some more information and call it "unsupported characters"

Reply:
I improved the error message.